app/code/community/Reverb/ReverbSync/Model/Sync/Order/Update.php

```diff
@@ -61,7 +67,8 @@ protected function _executeStatusUpdate($magento_order_entity_id, $reverb_order_
             // Fire event to allow for executing functionality upon order cancels/refunds
             Mage::dispatchEvent($event_name,
                                     array('order_entity_id' => $magento_order_entity_id,
-                                          'reverb_order_status' => $reverb_order_status)
+                                          'reverb_order_status' => $reverb_order_status,
+                                          'reverb_order_update_arguments_object' => $argumentsObject)
             );
```

Comment:
That's probably a good idea, yes. As long as it is understood that "reverb_order_update" is fired even though none of the data might have changed (since we're polling on an orders endpoint with overlapping time frames).